connectionmanager.js

```diff
@@ -1287,7 +1302,8 @@ export default class ConnectionManager {
 
             params.embyUserName = apiClient.getCurrentUserName();
 
-            if (apiClient.getCurrentUserId().toLowerCase() === '81f53802ea0247ad80618f55d9b4ec3c' && params.serverId.toLowerCase() === '21585256623b4beeb26d5d3b09dec0ac') {
+            const currentUserId = apiClient.getCurrentUserId();
+            if (currentUserId && currentUserId.toLowerCase() === '81f53802ea0247ad80618f55d9b4ec3c' && params.serverId.toLowerCase() === '21585256623b4beeb26d5d3b09dec0ac') {
```

Comment:
wtf is this?

Reply:
Looks like they blocked a specific user id and server id for some reason? This change only adds a check for `api.getCurrentUserId()` returning `null` it seems.

This is dealing with connect so it should be ripped out in the future also.